jax/ops/scatter.py

```diff
@@ -267,3 +270,19 @@ def segment_sum(data, segment_ids, num_segments=None):
   out = np.zeros((num_segments,) + data.shape[1:], dtype=data.dtype)
   segment_ids = np.mod(segment_ids, num_segments)
   return index_add(out, segment_ids, data)
+
+
+def index_by_mask(x, boolean_mask, axis=0, pad_value=0):
+  """A version of NumPy's `x[boolean_mask]`, but with a static output shape.
```

Comment:
This is restructured text, not markdown, and I think the backticks need to be doubled or written something like:
```
:code:`x[boolean_mask]`
```